exercise-4.js

```diff
@@ -0,0 +1,27 @@
+//0
+let apples = 20;
+let oranges = 30;
+//1
+console.log("1." + (apples == oranges));
+//2
+console.log("2." + (apples != oranges));
+//3
+console.log("3." + (apples > oranges));
+//4
+console.log("4." + (apples < oranges));
+//5
+console.log("5." + (oranges > apples));
+//6
+let mangoes = 5;
+console.log("6." + (mangoes * apples > mangoes + oranges));
```

Comment:
console.log((mangoes * apples) > (mangoes + oranges));

same with 7, 9 and 10

Reply:
It doesnt change anything